src/screens/VisualTests/NoBuild.tsx

```diff
@@ -22,59 +35,87 @@ interface NoBuildProps {
 }
 
 export const NoBuild = ({
-  error,
+  queryError,
   hasData,
   hasSelectedBuild,
   startDevBuild,
   localBuildProgress,
   branch,
   setAccessToken,
-}: NoBuildProps) => (
-  <Sections>
-    <Section grow>
-      {error && (
-        <Row>
-          <Col>
-            <Text>{error.message}</Text>
-          </Col>
-        </Row>
-      )}
+}: NoBuildProps) => {
+  const button = (
+    <Button small secondary onClick={startDevBuild}>
+      <Icons icon="play" />
+      Take snapshots
+    </Button>
+  );
+
+  let contents;
+  if (localBuildProgress) {
+    if (localBuildProgress.currentStep === "error") {
+      const firstError = Array.isArray(localBuildProgress.originalError)
+        ? localBuildProgress.originalError[0]
+        : localBuildProgress.originalError;
+      contents = (
+        <>
+          <ErrorContainer>
+            <b>Build failed:</b> <code>{firstError?.message || "Unknown Error"}</code>{" "}
```

Comment:
Just to be sure, we want the originalError instead of the formattedError so that users see more information?